docs/prover/cli/options.md

```diff
@@ -671,8 +671,14 @@ Prover.  `--settings` should be followed by a comma-separated list of options.
 This list is incomplete.
 ```
 
+(-optimisticFallback)=
+#### `--settings -optimisticFallback=true`
+
+This option determines whether to optimistically assume unresolved fallback 
```

Comment:
What exactly is meant by "fallback functions" here? As I'm familiar with the term in Solidity, the `fallback` function is called when the contract receives a function call whose `calldata` does not map to any of the contract's functions. And which contract is this applying to - the `currentContract` or external calls other contracts?

Reply:
Thanks for your question, I updated the description to better explain the effects of this setting. In short, this setting makes unresolved state mutating calls havoc only external contracts. External to the contract the function is called from, so not necessarily `currentContract`.